components/molecule/badgeCounter/src/styles/settings.scss

```diff
@@ -30,6 +30,10 @@ $molecule-badge-counter-large-offset-right: -($sz-base + $sz-base * 0.5) !defaul
 $fz-molecule-badge-counter-large: $fz-xs !default;
 $p-molecule-badge-counter-large: 0 $p-base !default;
 
+/* threeCharacters */
+$bdrs-molecule-badge-counter-threeCharacters: $bdrs-xxxl !default;
+$molecule-badge-counter-threeCharacters-diameter: 36px !default;
```

Comment:
haave you think about teh possibilityb to make it dependant to a `ch` unit?

Reply:
https://caniuse.com/?search=ch

Reply:
Not available for IE11 😭

Reply:
IE supports the ch unit, but unlike other browsers its width is that specifically of the "0" glyph, not its surrounding space. As a result, 3ch for example is shorter than the width of the string "000" in IE.

Reply:
Do we still support IE 11?